_includes/pbc/all/install-features/202404.0/install-the-cart-notes-feature.md

```diff
@@ -18,7 +18,7 @@ To start feature integration, integrate the required feature:
 ## 1) Install the required modules
 
 ```bash
-composer require spryker-feature/cart-notes:"{{page.version}}" --update-with-dependencies
+composer require spryker-feature/cart-notes-backend-api:"{{page.version}}" --update-with-dependencies
```

Comment:
https://github.com/spryker-feature/cart-notes-backend-api - 404

Reply:
All deprecated guides should be not available for a latest tag, so it's smells like an infrastructural issue in the documentation.

Reply:
oops, didn't check this part as well..